src/coreclr/src/jit/lclvars.cpp

```diff
@@ -2369,6 +2369,35 @@ void Compiler::lvaSetVarAddrExposed(unsigned varNum)
     lvaSetVarDoNotEnregister(varNum DEBUGARG(DNER_AddrExposed));
 }
 
+//------------------------------------------------------------------------
+// lvaSetVarLiveInOutOfHandler: Set the local varNum as being live in and/or out of a handler
+//
+// Arguments:
+//    varNum - the varNum of the local
+//
+void Compiler::lvaSetVarLiveInOutOfHandler(unsigned varNum)
+{
+    noway_assert(varNum < lvaCount);
```

Comment:
Do you really need a "noway" assert here? Otherwise using `lvaGetDesc` below would provide a normal assert.

Reply:
Probably not; not sure why I put that in as it.